pkg/rootless/rootless.go

```diff
@@ -65,9 +63,6 @@ func Rootless(stateDir string) error {
 	}
 
 	os.Setenv(childEnv, filepath.Join(parentOpt.StateDir, parent.StateFileAPISock))
-	if parentOpt.EvacuateCgroup2 != "" {
-		os.Setenv(evacuateCgroup2Env, "1")
```

Comment:
`evacuateCgroup2Env = "_K3S_ROOTLESS_EVACUATE_CGROUP2"` on line 26 should be dropped as well